src/layman/layer/rest_test.py

```diff
@@ -166,16 +166,17 @@ def test_layman_gs_user_conflict(client):
             fp[0].close()
 
 
-@pytest.mark.usefixtures('app_context')
 def test_wrong_value_of_layername(client):
-    username = 'testuser1'
+    username = 'test_wrong_value_of_layername_user'
+    layername = 'layer1'
+    publish_layer(username, layername, client)
+    delete_layer(username, layername, client)
```

Comment:
This would, in my opinion, deserve comment.

Reply:
Good point